dotCMS/src/main/java/com/dotcms/experiments/model/AbstractExperiment.java

```diff
@@ -88,6 +88,12 @@ default Instant modDate() {
     @JsonProperty("targetingConditions")
     Optional<List<TargetingCondition>> targetingConditions();
 
+    @JsonProperty("loopbackWindow")
+    @Value.Default
+    default int loopbackWindow() {
+        return 10;
```

Comment:
not sure if this should be hardcoded

Reply:
It's the way to specify a default value in Immutables org